examples/pipelines/starcoder/components/filter_comments

```diff
@@ -0,0 +1 @@
+/Users/nielsrogge/Documents/python_projects/fondant/components/filter_comments
```

Comment:
can be removed

Reply:
I needed the symbolic links in order to run the pipeline locally

Reply:
Only with the script right. I would not include this in git. The local runner will solve this.